attributes/azure.rb

```diff
@@ -21,38 +15,28 @@
   'free_node_count' => 5,
 }
 
+creds = Chef::DataBagItem.load('marketplace_image', 'publishing_credentials')
+
 azure_builder_config = {
   'name' => 'azure_automate_BYOL',
-  'type' => 'azure',
-  'publish_settings_path' => publish_settings_path,
-  'subscription_name' => 'Partner Engineering',
-  'storage_account' => 'ampimages',
-  'storage_account_container' => 'images',
+  'type' => 'azure-arm',
+  'subscription_id' => "#{creds['azure']['subscription_id']}",
+  'client_id' => "#{creds['azure']['client_id']}",
+  'client_secret' => "#{creds['azure']['client_secret']}",
+  'resource_group_name' => 'publish-marketplace-images',
+  'storage_account' => 'marketplaceimages',
+  'capture_container_name' => 'vhds',
   'os_type' => 'Linux',
-  'os_image_label' => 'Ubuntu Server 14.04 LTS',
+  'image_publisher' => 'Canonical',
+  'image_offer' => 'UbuntuServer',
+  'image_sku' => '14.04.5-LTS',
   'location' => 'East US',
-  'instance_size' => 'Large',
-  'user_image_label' => 'Chef_Automate_BYOL_{{timestamp}}',
+  'vm_size' => 'Standard_D3_v2',
+  'capture_name_prefix' => 'Chef_Automate_BYOL',
```

Comment:
Does the builder automatically append a timestamp to the built VHD?

Reply:
It doesn't: vhds have names like `Chef_Automate_BYOL-osDisk.88a0957e-f746-4baa-9436-6eeb39bfa859.vhd`.

I originally tried to add a timestamp and got the following error because the prefix was too long:

```
A capture_name_prefix must satisfy the regular expression "^[A-Za-z0-9][A-Za-z0-9_\\-\\.]{0,23}$".
```